build/podman/src/test/java/io/quarkus/ts/docker/PodmanBuildIT.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.ts.docker;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import com.github.dockerjava.api.model.Image;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.utils.DockerUtils;
+
+@QuarkusScenario
+@EnabledIfEnvironmentVariable(named = "DOCKER_HOST", matches = ".*podman.*")
+/**
+ * We run the tests only when podman is enabled and does not pretend to be Docker
+ * since otherwise we won't be able to distinguish between the extension being broken
+ * and a situation, when podman is not installed.
+ *
+ * Unfortunately, that means, that the test is disabled on Windows and Mac
```

Comment:
This feels clumsy, why don't you try `podman` cli command instead, or are there no Podman specific environment variables or something else? We have Windows tests with Podman, so this must be tested there, or not?

Reply:
oki